pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, pandoc
+, esbuild
+, deno
+, fetchurl
+, makeWrapper
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v${finalAttrs.version}/quarto-${finalAttrs.version}-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
+
+  buildInputs = [
+      pandoc
+      deno
+      esbuild
+      makeWrapper
+  ];
+
+  patches = [
+      ./fix-deno-path.patch
+  ];
+
+  preFixup = ''
+      wrapProgram $out/bin/quarto --prefix PATH : ${lib.makeBinPath [ deno ]};
```

Comment:
Shouldn't the other binaries in `bin/tool` be also added to PATH? `pandoc` for sure, I'd say.

Reply:
The above suggestion is wrong because it quadruple wraps the program and we can set all of this in one wrapProgram call since it accepts multiple prefix's